frontend/src/components/menu/index.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { ReactNode } from 'react';
+import { ListGroup, Form } from 'react-bootstrap';
+import classNames from 'classnames';
+import ClickAwayListener from 'react-click-away-listener';
+import './menu.scss';
+
+export interface IMenuItemProps {
```

Comment:
I think we should start adding documentation to our exports.  It's tech-debt that is quickly adding up.